### PR TITLE
New post: Getting started with Graphcore and Optimum

### PR DESCRIPTION
**Pull request by juliensimon**
Guest post from Graphcore DevRel

### CODE REVIEW

File: graphcore-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+<div class="blog-metadata">
+    <small>Published November 30, 2021.</small>
+    <a target="_blank" class="btn no-underline text-sm mb-5 font-sans" href="https://github.com/juliensimon/blog/blob/master/getting-started-graphcore.md">
+        Update on GitHub
+    </a>
+</div>
+
+<div class="author-card">
+    <a href="https://twitter.com/internetoftim">
```
Comment (julien-c) [inline graphcore-getting-started.md:19]: we should link to a profile on-platform (i.e. a hf.co username or org) cc @osanseviero

File: graphcore-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        <div class="bfc">
+            <code> internetoftim </code>
```
Comment (julien-c) [inline graphcore-getting-started.md:21]: we should mark it explicitly as `guest` blogpost, we have some markup in other posts for this IIRC

File: graphcore-getting-started.md (hunk 1 above)
Comment (julien-c) [reply graphcore-getting-started.md:19]: they are at https://huggingface.co/internetoftim from what i can see

File: graphcore-getting-started.md (hunk 2 above)
Comment (osanseviero) [reply graphcore-getting-started.md:21]: YEs, we can add `<span class="bg-gray-100 rounded px-1 text-gray-600 text-sm font-mono">guest</span>` after the full name

**Comment by feconroses** [comment]
@julien-c @osanseviero added a new commit where I added author images + added a link to Tim's HF profile + added `guest` tag. Could we please merge and deploy so we can launch? 🙏

**Comment by feconroses** [comment]
FYI we just committed the new metrics and the changes discussed by Julien/Omar (e.g. adding a link to Tim’s HF user + adding ‘guest’ tag). We can now merge and deploy.

File: graphcore-getting-started.md (hunk 2 above)
Comment (feconroses) [reply graphcore-getting-started.md:15]: sounds good! We added the `guest` tag in the latest commit

File: graphcore-getting-started.md (hunk 1 above)
Comment (feconroses) [reply graphcore-getting-started.md]: Added a link to Timás HF profile in the latest commit

**Pull request merged**